packages/cozy-harvest-lib/src/components/DeleteAccountButton.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import omit from 'lodash/omit'
+
+import { Button } from 'cozy-ui/react/Button'
+import { translate } from 'cozy-ui/react/I18n'
+import { withMutations } from 'cozy-client'
+
+import accountsMutations from '../connections/accounts'
+
+export class DeleteAccountButton extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loggingOut: false
+    }
+    this.handleDelete = this.handleDelete.bind(this)
+  }
+
+  async handleDelete() {
+    this.setState({ loggingOut: true })
+    const { account, deleteAccount, onError, onSuccess } = this.props
+    try {
+      await deleteAccount(account)
+      if (typeof onSuccess === 'function') onSuccess(account)
+    } catch (e) {
+      if (typeof onError === 'function') {
+        onError(e)
+      } else {
+        // eslint-disable-next-line no-console
+        console.error(e)
+      }
+    } finally {
+      this.setState({ loggingOut: false })
+    }
+  }
+
+  render() {
+    const { t } = this.props
+    const { loggingOut } = this.state
+    return (
+      <Button
+        theme="danger-outline"
+        disabled={loggingOut}
+        busy={loggingOut}
+        onClick={this.handleDelete}
+        label={t('accountForm.disconnect.button')}
+        {...omit(this.props, Object.keys(DeleteAccountButton.propTypes))}
+      />
+    )
+  }
+}
+
+DeleteAccountButton.propTypes = {
+  /**
+   * io.cozy.accounts document
+   */
+  account: PropTypes.object.isRequired,
```

Comment:
Tip: You can use the shorter

```
  /** @type {[type]} [description] */
```

I get these when I type `/**` `[space]` in my editor.

Reply:
Thanks for the tip, for now I'll keep like this to be consistent with other components